deployment/hasura/metadata/actions.graphql

```diff
@@ -132,6 +132,13 @@ type Query {
   ): SequenceSeqJson!
 }
 
+type Query {
+  generateSequenceSeqJsonEDSL(
```

Comment:
I'm wondering if we can call this action `getUserSequenceSeqJson`?
And maybe later in a separate PR we can rename the exisiting `getSequenceSeqJson` to `getExpansionSequenceSeqJson`? This way each type is more explicit and paves the way for keeping both sequence types in the database.

Reply:
I thought about this as well when reviewing. I just wasn't sure what the most obvious names would be. In both cases, we are generating SeqJson from the EDSL, but in one, we take a Sequence EDSL, in the other, we generate from sequence constructs in the plan.

Reply:
Sure I can make this change. Yeah, coming up with a name is tricky as I don't want to make it super long, but also want it to be meaningful :) I will also rebase with develop - I saw your slack message @camargo .